includes/admin/class-wc-stripe-payment-gateways-controller.php

```
@@ -77,6 +77,15 @@ public function enqueue_payments_scripts() {
 		);
 
 		if ( $is_payment_methods_page ) {
+			$gateway = WC_Stripe::get_instance()->get_main_stripe_gateway();
+			wp_localize_script(
+				'woocommerce_stripe_payment_gateways_page',
+				'wc_stripe_settings_params',
+				[
+					'show_stripe_first_method_notice' => WC_Stripe_Helper::should_show_stripe_first_method_notice(),
```

PHPMD (2.15.0)

[error] Avoid using static access to class 'WC_Stripe_Helper' in method 'enqueue_payments_scripts'. (undefined)

(StaticAccess)